renderer/reducers/createReducer.js

```diff
@@ -0,0 +1,10 @@
+import produce from 'immer'
+
+const createReducer = (initialState, actionHandlers) => (state = initialState, action) => {
```

Comment:
Lets move this into `reducers/utils` or something like that

Reply:
The reason why I went with a separate file was to avoid cyclical deps, since `utils` use reducers a lot.

Reply:
I agree that it should be in a separate file - but thought it made sense to place that file in a subdir so that it's separate from all of the actual reducers

Reply:
got it. i'll update

Reply:
Oh lol, I forgot we actually have `reducers/utils.js` already

Reply:
yeah..

Reply:
@mrfelton i've moved createReducer to `reducers/utils/createReducer.js`

Reply:
oops, rebased against master by mistake. fixed